config/pipeline.yaml

```diff
@@ -0,0 +1,200 @@
+
+# Configuration for running the complete pipeline including modeling and evaluation.
+
+# Absolute paths to non-code resources.
+file_paths:
+  queries: "/home/jupyter/github_msba_capstone/msba-capstone-2020/queries"
+
+# Global config used by multiple pipeline steps. Also made available
+# as parameters to all SQL queries.
+global:
+  # Project and dataset configuration.
+  source_project_id: "bigquery-public-data"
+  destination_project_id: "ut-goog"
+  source_dataset: "cfpb_complaints"
+  destination_dataset: "cfpb_complaints"
+
+  # Clean data, minor refined versions of provided data.
+  source_table: "complaint_database"
+  nlp_features_table: "complaints_nlp"
+  cleaned_features_table: "complaints_clean_categories"
+  nulls_removed_table: "complaints_nulls_removed"
+  clean_table: "complaints_clean"
+  
+  # Split features into training and prediction.
+  train_predict_split: "TrainPredictSplit"
+  features_train_table: "FeaturesTrain"
+  features_predict_table: "FeaturesPredict"
+
+  # Output tables for predictions and failed predictions from the batch
+  # prediction job, ex. numeric column recieved a string.
+  predictions_table: "Predictions"
+  failed_predictions_table: "FailedPredictions"
+  
+  # Evaluation
+  eval_table: "Evaluation"
+  eval_metrics_table: "EvaluationMetrics"
+
+  # AutoML Parameters. It is assumed that no other datasets or models have
+  # been created with these names, or the pipeline will fail. They can be
+  # deleted using the client, or the UI.
+  dataset_display_name: "CFPBDataset"
+  model_display_name: "CFPBModel"
+  automl_compute_region: "us-central1"
+
+  # Final model will be trained on test data, metrics are currently based
+  # on the predict split instead. AutoML Requires at least one row of test data
+
+# Files with templated queries, in file_paths.queries.
+query_files:
+  remove_nulls: "remove_nulls.sql"
+  clean_categories: "clean_categories.sql"
+  combine_tables: "combine_tables.sql"
+  train_predict_split: "train_and_predict_split.sql"
+  prediction_features: "prediction_features.sql"
+  training_features: "training_features.sql"
+
+# Parameters specific to individual SQL pipeline steps.
+query_params:
+
+  removing_nulls:
+    column1: "consumer_complaint_narrative"
+    column2: "company_response_to_consumer"
+    
+  category_cleanup:
+    old_issue1: "Dealing with your lender or servicer"
+    old_issue2: "Dealing with my lender or servicer"
+    new_issue1: "Other"
+    new_issue2: "Dealing with lender or servicer"
+    old_subprod1: "Other (i.e. phone, health club, etc.)"
+    old_subprod2: "Loan"
+    new_subprod1: "Other"
+    new_subprod2: "Other Loan"
+    old_subissue1: "Debt is not yours"
+    old_subissue2: "Debt is not mine"
+    old_subissue3: "Account status"
+    new_subissue1: "Other"
+    new_subissue2: "Incorrect debt"
+    new_subissue3: "Account status incorrect"
+    
+  train_predict_split:
```

Comment:
It would be better to code this as like "test percentage". and then make the train threshold 1-test in the query.

Reply:
Done this.